<meta charset="UTF-8">
<title>A New Tool for Predicting Long-distance Target Genes of Regulatory Mutations by Integrating Multi-omics Data with a case Study in Follicular Lymphoma</title>
<link rel="stylesheet" type="text/css" href="styles.css"> 
</head>
<body>
<header>
    <div class="logo">
        <a style="text-decoration:none;" href ="https://www.ahus.no/"><img src="figures/ahhus.png",alt="ahus"></a>
        <a style="text-decoration:none;" href ="https://www.helse-sorost.no/"><img src="figures/helse-sor-ost.jpg",alt="helse sorost"></a>
        <a style="text-decoration:none;" href="https://www.oslo-universitetssykehus.no/"><img src="figures/oslo-universitetssykehus-ous-logo.png",alt="oslo univeristetsykehus"></a>
        <a style="text-decoration:none;" href="https://www.uio.no/"><img src="figures/uio_segl_a.png" alt="oslo Universitet"></a>
    </div>
    <h1>Differential Network Analysis in Intra-chromosomal Community Interactions for MCF7 Breast Cancer System</h1>
    <nav>
        <ul>
            <li><a href=""> link to documentation</a></li>
            <li><a href="">link to github</a></li>
            <li><a href="">link to article </a></li>
        
        </ul>
    </nav>
</header>



    <section class="authors">
        <h2 class=> <strong> Authors:</strong></h2>
        <p >
            <br>
           <strong> Zhihao Yao <sup>1,5</sup>, Kun Fang <sup>2,3,4</sup>, Jenny Sofie Dragland <sup>6</sup>, Magnar Bjørås <sup>5,7</sup>, Victor X. Jin <sup>2,3,4</sup>, Junbai Wang <sup>1*</sup></strong>
           <br>
            <br>1. Department of Clinical Molecular Biology (EpiGen), Akershus University Hospital and University of Oslo, Lørenskog, Norway
            <br>
            <br>2. Division of Biostatistics, Medical College of Wisconsin, Milwaukee, WI 53226, USA
            <br>
            <br>3. MCW Cancer Center, Medical College of Wisconsin, Milwaukee, WI 53226, USA
            <br>
            <br>4. Mellowes Center for Genomic Sciences and Precision Medicine, Medical College of Wisconsin, Milwaukee, WI 53226, USA
            <br>
            <br>5. Department of Microbiology, Oslo University Hospital and University of Oslo, Oslo, Norway 
            <br>
            <br>6. Department of Pathology, Oslo University Hospital - Norwegian Radium Hospital, Oslo, Norway
            <br>
            <br>7. Department of Clinical and Molecular Medicine, Norwegian University of Science and Technology, Trondheim, Norway
            <br>
            <br>
            <br>
            <em>
                <br>
                <br>* To whom correspondence should be addressed. Email: junbai.wang@medisin.uio.no
            </em>
        </p>
    </section>



    <section class="abstract">
        <h2 class="center-align">Abstract </h2>
         <p class="centered-line-spacing"> <br>  
            Extracting critical information from intra-chromosomal interaction dynamics is challenging, especially when integrated with high-dimensional epigenetic modifications.
             A novel computational pipeline is proposed to identify differential intra-chromosomal interactions by integrating Hi-C data with other genomic features, and two novel parameter tuning methods enable network clustering method to identify valid intra-chromosomal communities at different interaction resolutions.
              Within the validated communities, enrichment tests of genomic features in zero, one hour estrogen (E2)-treated MCF7 cells, and tamoxifen-resistant MCF7TR cells reveal two major types of communities - active/repressive compartments, respectively, which resembles the properties of open/closed chromatin in the genome.
               By incorporating Hi-C topological features into genomic features enrichment in the communities, differential network analysis of MCF7 and tamoxifen-resistant MCF7TR breast cancer systems suggests 515 genes with their differential sub-networks are significantly associated with both differential interactions and differential expressions (DIDEs).
              The looping-mediated mechanisms disclosed by these DIDEs are consistent with known biological perspectives.
             It demonstrates that the proposed computational pipeline has strong positive impact on identifying biologically meaningful interaction changes based on the integrative data analysis of chromosomal interactions with other genomic features.
            <br>
        </section>

    <section class="supplementary">
        <h2 class="center-align">Supplementary Files </h2>

            <br>
            <p><a href="supplementary_files/suppl.1.selected_diff_inter_nodes_pval_ls0.05.xlsx" class="download-link" download>Supplementary File 1.</a>  - Differential interacting nodes between MCF7 cells and tamoxifen-resistant MCF7TR cells at 50kb resolution.<br>
            The Euclidean distances evaluating topological and genomic features of all network nodes are fitted to a Gaussian distribution first. The 2681 differential interacting nodes are screened out through p-value < 0.05. Each node is a window bin with size 50kb.</p>
            <p><a href="supplementary_files/suppl.2.diff_inter_genes_in_selected_nodes.xlsx" class="download-link" download> Supplementary File 2. </a> - Differential interacting genes between MCF7 cells and tamoxifen-resistant MCF7TR cells at 50kb resolution <br>
                1378 genes and the chromosomes to which they belong. Gene symbols are determined according to the node names in Supplementary file 1 (The name of each node represents its position in the chromosome to which it belongs). </p>
            <p><a href="supplementary_files/suppl.3.selected_515_DIEGs_50kb_resolution.xlsx" class="download-link" download>Supplementary File 3.</a> - Differential interacting and Differentially expressed genes (DIEGs) between MCF7 cells and tamoxifen-resistant MCF7TR cells at 50kb resolution <br>
                515 differentially expressed genes (DEGs) (absolute(relative ratio) > 0.66) are selected from the 1378 differential interacting genes. </p>
            <p><a href="supplementary_files/suppl.4.david_for_515_genes.xlsx"  class="download-link"  download>Supplementary File 4. </a> - Gene ontology (GO) and biological pathway enrichment test based on the 515 DIEGs by using DAVID tool <br>
                Enriched functional annotations from REACTOME pathway, KEGG pathways, GO molecular function (MF), and GO biological process (BP) are selected under the thresholds of Fold Enrichment >= 2 and p-value =< 0.05. Functional annotations with the top 15 Fold Enrichments are highlighted in colors. </p>
            <p><a href="supplementary_files/suppl.5.feature_in_DIEGs_nodes_untreated.xlsx" class="download-link"  download>Supplementary File 5. </a> - Network nodes and genomics features of the 515 DIEGs under MCF7 cells <br>
                Network nodes and communities where the 515 DIEGs located are identified under MCF7 cells. Minimal and maximal values of enhancer markers (h3k4me1, h3k4me3, and h3k27ac), repressor markers (h3k27me3 and h3k9me3), and gene expression within each node are recorded.</p>
            <p><a href="supplementary_files/suppl.6.feature_in_DIEGs_nodes_tamr.xlsx" class="download-link"  download>Supplementary File 6. </a>- Network nodes and genomics features of the 515 DIEGs under tamoxifen-resistant MCF7TR cells <br>
                Network nodes and communities where the 515 DIEGs located are identified under tamoxifen-resistant MCF7TR cells. Minimal and maximal values of enhancer markers (h3k4me1, h3k4me3, and h3k27ac), repressor markers (h3k27me3 and h3k9me3), and gene expression within each node are recorded.</p>
            <p><a href="supplementary_files/suppl.7.network_for_DIEGs_nodes_untreated.xlsx" class="download-link"  download>Supplementary File 7. </a>- Matrices of differential sub-networks under MCF7 cells <br>
                Each node is a 50kb chromosome segment containing DIEGs. The value of the edge represents whether there is a significant intra-chromosomal interaction between two nodes, where 1 represents an interaction and 0 represents no interaction. </p>
            <p><a href="supplementary_files/suppl.8.network_matrix_for_DIEGs_nodes_tamr.xlsx" class="download-link"  download>Supplementary File 8.</a>- Matrices of differential sub-networks under tamoxifen-resistant MCF7TR cells. <br>
                Each node is a 50kb chromosome segment containing DIEGs. The value of the edge represents whether there is a significant intra-chromosomal interaction between two nodes, where 1 represents an interaction and 0 represents no interaction. </p>
            <p><a href="supplementary_files/suppl.9.differential_sub_networks.pdf" class="download-link" download>Supplementary File 9. </a> - Figures of differential sub-networks between MCF7 cells and tamoxifen-resistant MCF7TR cells at 50kb resolution <br>
                The differential sub-networks are composed of selected nodes and significant Hi-C intra-chromosomal interactions between them, where the selected nodes are identified based on a differential network analysis incorporating topological and genomic features between MCF7 cells and tamoxifen-resistant MCF7TR cells at 50kb resolution (p-value < 0.05 for differential interacting nodes; absolute (relative ratio) > 0.66 for differentially expressed genes).
                 The orange and blue colors in the nodes represent whether the histone markers of the community are active or not. The colors of the outer rings represent previously identified intra-chromosomal communities.
                  Gray lines between nodes represent significant Hi-C intra-chromosomal interactions. All the figures are visualized by Cytoscape based on the Supplementary file 5-8.
            </p> 
            <p> <a href="supplementary_files/suppl.10.david_for_activated_community_in_chr4.xlsx" class="download-link" download> Supplementary File 10. </a>  Gene ontology (GO) and biological pathway enrichment test based on the genes from a sub-network in chromosome 4 (see Figure 9 in manuscript) <br>
                Gene ontology (GO) and biological pathway enrichment test results based on 25 DIEGs genes on chromosome 4. These genes are enriched in one activated community under MCF7 condition (left panel of Figure 9C), but in two repressed communities under MCF7TR condition (right panel of Figure 9C). </p>

    </section>



    <footer>
        <p> For correnspondance please contact Junbai Wang, Email: junbai.wang@medisin.uio.no </p>
    </footer>
</body>
</html>